core/jobs/batch_jobs/skill_migration_jobs.py

```diff
@@ -127,85 +132,25 @@ def _generate_skill_changes(
             })
             yield (skill_id, skill_change)
 
-    @staticmethod
-    def _update_skill(
-        skill_model: skill_models.SkillModel,
-        migrated_skill: skill_domain.Skill,
-        skill_changes: Sequence[skill_domain.SkillChange]
-    ) -> Sequence[base_models.BaseModel]:
-        """Generates newly updated skill models.
+    def expand(
+        self, pipeline: beam.Pipeline
+    ) -> Tuple[
+        beam.PCollection[base_models.BaseModel],
+        beam.PCollection[job_run_result.JobRunResult]
+    ]:
+        """Migrate skill objects and flush the input
+            in case of errors.
```

Comment:
Ditto, please fix the wrapping here and elsewhere. I think some words (if not all) can be moved to the previous line and still fit within the 80-char limit.

Ditto elsewhere for other docstrings too. Thanks!